instrumentation/opentelemetry-instrumentation-urllib3/src/opentelemetry/instrumentation/urllib3/__init__.py

```diff
@@ -83,15 +83,19 @@ def response_hook(
 ---
 """
 
+from __future__ import annotations
+
 import collections.abc
 import io
 import typing
 from dataclasses import dataclass
 from timeit import default_timer
-from typing import Collection
+from typing import Any, Callable, Collection, Mapping, Optional
 
 import urllib3.connectionpool
```

Comment:
Maybe update callers of this while we are it? It looks like we are only using `urllib3.connectionpool.HTTPConnectionPool`

Reply:
You mean the `wrap` calls? I didn't want to do it because it may have some runtime unexpected behavior that I'm not aware. Should I change it as well?